pkg/sync/http/http_sync_test.go

```diff
@@ -0,0 +1,158 @@
+package http
+
+import (
+	"context"
+	"io"
+	"log"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/open-feature/flagd/pkg/sync"
+
+	"github.com/golang/mock/gomock"
+	"github.com/open-feature/flagd/pkg/logger"
+	syncmock "github.com/open-feature/flagd/pkg/sync/http/mock"
+)
+
+func TestSimpleSync(t *testing.T) {
+	ctrl := gomock.NewController(t)
+
+	t.Run("SimpleSync", func(t *testing.T) {
```

Comment:
Is there a reason why this is executed in a subtest? Apologies if you haven't introduced this

Reply:
Thank you for catching this. I will convert this to a simple test case.  Same is true for file sync test